test/unitTests/core/test-totalSupplyMintKeeper.js

```diff
@@ -0,0 +1,29 @@
+import { test } from 'tape-promise/tape';
+import { makeMint } from '../../../core/issuers';
+import { makeTotalSupplyConfig } from '../../../core/config/totalSupplyConfig';
+
+test('totalSupplyConfig', async t => {
```

Comment:
We could simulate dropping a payment by calling split in a separate function, or at least a sub-scope, and letting part of it go out of scope. 

I realize that it won't change totalSupply, but it's still good to have that case explicitly represented, and show what we expect.

Reply:
Ok, I can simulate dropping the payment, and can show that the totalSupply doesn't change.  Since we can't know when garbage collection will happen, we can't include that part in the test, but it sounds like that's ok.